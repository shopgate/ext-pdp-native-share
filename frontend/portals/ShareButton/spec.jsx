import React from 'react';
import { mount } from 'enzyme';

// eslint-disable-next-line require-jsdoc
const MockedShareButton = () => (<div>ShareButton</div>);
jest.mock('../../components/ShareButton', () => MockedShareButton);

describe('ShareButton', () => {
  // eslint-disable-next-line global-require
  const ShareButton = require('./index').default;
  it('should render ShareButton', () => {
    const component = mount(<ShareButton />);
    expect(component.find(MockedShareButton).exists()).toBe(true);
    expect(component).toMatchSnapshot();
  });
});
