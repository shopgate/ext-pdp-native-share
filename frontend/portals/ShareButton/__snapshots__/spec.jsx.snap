// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShareButton should render ShareButton 1`] = `
<_default>
  <MockedShareButton>
    <div>
      ShareButton
    </div>
  </MockedShareButton>
</_default>
`;
